src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  position: relative;
+  display: flex;
+  align-items: center;
+
+}
+
+.logo {
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
```

Comment:
- The `header` shouldn't be relative and `nav` absolute. Just use flex, https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content.
- Don't use `margin-left` and absolute positioning `right: 50px;`, specify spaces with padding on the `header` element
- Don't leave unnecessary blank lines